content/docs/usage/sdks.md

```diff
@@ -3,4 +3,4 @@ title: OpenClarity SDKs
 weight: 5000
 ---
 
-See the [scanner module](https://pkg.go.dev/github.com/openclarity/vmclarity/scanner) for the SDK.
\ No newline at end of file
+See the [scanner module](https://pkg.go.dev/github.com/openclarity/openclarity/scanner) for the SDK.
```

Comment:
this link gives "Oops! We couldn't find “github.com/openclarity/openclarity/scanner”."

Reply:
This is okay as this link will only be available once we tag a release of OpenClarity